workspace_host/lib/docker.js

```diff
@@ -34,7 +34,7 @@ function parseDockerLogs(buffer) {
     return out;
   }
 
-  return output;
+  return Buffer.concat(outputChunks);
```

Comment:
Classic O(n^2) to O(n) fix. Nice!

Reply:
Yep! I almost want ESLint to yell at me if I call `Buffer.concat` in any kind of loop, this is the second or third time this has really bitten me!